generated/nifpga/nifpga_service.cpp

```diff
@@ -63,6 +63,29 @@ namespace nifpga_grpc {
     }
   }
 
+  //---------------------------------------------------------------------
+  //---------------------------------------------------------------------
+  ::grpc::Status NiFpgaService::AcknowledgeIrqs(::grpc::ServerContext* context, const AcknowledgeIrqsRequest* request, AcknowledgeIrqsResponse* response)
```

Comment:
Did you try to call these 2 APIs over gRPC and verify they work as expected? I didn't see anything about this in your review description.